tests/src/bin/precompute_shares.rs

```diff
@@ -11,13 +11,15 @@ use rand::{rngs::OsRng, CryptoRng, RngCore};
 fn main() -> Result<()> {
     match args() {
         Operation::GenShares => precompute_shares(),
+        Operation::GenOldShares { out_dir } => generate_old_share(&out_dir),
         Operation::GenPrimes => precompute_primes(),
     }
 }
 
 #[derive(Clone, Debug)]
 enum Operation {
     GenShares,
+    GenOldShares { out_dir: std::path::PathBuf },
     GenPrimes,
 }
```

Comment:
Add clippy ignore? I see it fails

Reply:
Oh I ignored CI because of failing publish workflow... I thought only publish-* was failing :sweat_smile:

Reply:
fixed